scripts/feature_tests/upgrade/upgrade_vars.sh

```diff
@@ -18,8 +18,5 @@ export CAPM3_REL_TO_VERSION
 export CAPIRELEASE="v0.3.15"
 export CAPI_REL_TO_VERSION="v0.3.16"
 
-export CAPI_API_VERSION="v1alpha3"
-export CAPM3_API_VERSION="v1alpha4"
```

Comment:
This variables are needed for upgrade test to build the environment. We shouldn't delete these variables.

Reply:
Now its passing the CI run because we are using by default capi v1a3 and capm3 v1alpha4. But when we introduce the change for capi v1a4 it will break.

Reply:
So these are specifically only here for testing the upgrade? Could we then maybe rename them to something like `CAPI_UPGRADE_VERSION` or `CAPI_OLD_VERSION`? As it is now I think it is very confusing with both `CAPI_VERSION` and `CAPI_API_VERSION`. :thinking:

Reply:
@Xenwar can you please have a look in this change?
/cc @Xenwar

Reply:
Added a comment

Reply:
@Xenwar This is supposed to be resolved I think ? I cannot find your comment regarding this